prompts/prompts.go

```diff
@@ -64,31 +60,28 @@ func SelectProduct(products []*cModels.Product, label string) (int, string, erro
 			return 0, "", errs.ErrProductNotFound
 		}
 
-		fmt.Println(promptui.SuccessfulValue("Product", line(products[idx])))
+		fmt.Println(promptui.SuccessfulValue("Product", label)) // FIXME
```

Comment:
What's broken?

Reply:
`promptui.SuccessfulValue` doesn't return the same style anymore. I added to the todo list to fit it.